templates/README.md

```diff
@@ -18,16 +18,13 @@ _Note_: Maintainers will be emailed if a release fails.
 
 ```jsonc
 {
-  "homepage": "YOUR_HOMEPAGE", // <-- Edit this
+  "homepage": "INSERT_YOUR_HOMEPAGE", // <-- Edit this
   "maintainers": [
     {
       // <-- And this
-      "name": "YOUR_NAME",
-      "email": "YOUR_EMAIL",
-      "github": "YOUR_GITHUB_USERNAME",
-      // To find your numeric user ID:
-      // curl https://api.github.com/users/YOUR_USERNAME | jq .id
-      "github_user_id": "YOUR_GITHUB_ID_NUMBER"
```

Comment:
We'll want to keep this.

Reply:
The app does it automatically, but it's also good to be intentional about the id.

Reply:
oops, this was a bad merge